README.md

```diff
@@ -2,4 +2,6 @@
 
 VSTeam Quickstart Scripts
 
-This repo contains all currently available VSTeam scripts contributed by the community.
\ No newline at end of file
+This repo contains all currently available VSTeam scripts contributed by the community.
+
+See the [**Contribution guide**](/CONTRIBUTING.md#contribution-guide) for how to use or contribute to this repo.
```

Comment:
Shall we also add the statement about MIT license and that this is also using the [Contributor Covenant Code of Conduct](https://github.com/MethodsAndPractices/vsteam/blob/master/CODE_OF_CONDUCT.md)?